### PR TITLE
Add venue details page

### PR DESCRIPTION
**Pull request by nikomakela**
LIIKUNTA-336. Migrated the old Liikunta-Helsinki venue details page components to fit in their new context inside the monorepo. Tried to use some common components and not to copy components that would duplicate any features.

NOTE: Testing this PR before the #109 and #110 are deployed needs a local Apollo Router, because otherwise the Venue-API won't be available!

Add the next line in the `.env.local`

```NEXT_PUBLIC_FEDERATION_ROUTER_ENDPOINT="http://localhost:4000"```

and run the Apollo Router locally:

```yarn docker:graphql-router:sports:serve --build```

image

image

----
Multi-step deployment, because the current CI environment cannot handle a concurrent deployment for router and app changes:

1. Copy and migrate the Apollo-Server for Venues and Hauki data from the old deprecated Liikunte-Helsinki repo: https://github.com/City-of-Helsinki/events-helsinki-monorepo/pull/110
2. Do the Apollo Router work for the added Venue-subgraph (added in step 1): https://github.com/City-of-Helsinki/events-helsinki-monorepo/pull/109
3. Add (copy and migrate) the Venue details page when the Venue datasource is available (added in step 2): https://github.com/City-of-Helsinki/events-helsinki-monorepo/pull/108

### CODE REVIEW

File: apps/sports-helsinki/src/domain/graphql/createApolloServer.ts [hunk 1]
```diff
@@ -20,15 +22,15 @@ const isProd = process.env.NODE_ENV === 'production';
...
-const initQueryTypeDefs = gql`
-  type Query {
-    _empty: String
-  }
-`;
+// const initQueryTypeDefs = gql`
+//   type Query {
+//     _empty: String
+//   }
+// `;
```
[the remark this answers is not in the document]
Comment (nikomakela) [reply apps/sports-helsinki/src/domain/graphql/createApolloServer.ts:29]: I guess it has been a test whether it's needed still or not. Seems like not!

**Review by tuomas777 — APPROVED** [review]
clap

**Comment by terovirtanen** [comment]
**Sports-Helsinki branch is deployed to platta: https://liikunta-pr108.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**Hobbies-Helsinki branch is deployed to platta: https://harrastukset-pr108.dev.hel.ninja** :rocket::rocket::rocket:

**Pull request merged**